### DISCUSSION TITLE
@nxext/stencil w/ NX 16

### DISCUSSION THREAD

**Discussion by wederchr** [nxext/nx-extensions#945] [General]
I saw that as part of migrating the @nxext/* plugins to NX 16 @nxext/stencil has been temporarily removed but there is a PR to re-add it: https://github.com/nxext/nx-extensions/pull/935.

Do you have an ETA for the next release that includes the @nxext/stencil plugin? Is there anything I can help with/ contribute to?

**Comment by DominikPieper** [comment]
Hey, no ETA so far. Most of the AST stuff around the outputtarget generators have to be refactored cause the Nx stuff I used there is not there anymore or in other packages. That’s something that needs a fix before

**Reply by wederchr** [reply]
Hey, any update on this?

**Comment by mathpaquette** [comment]
Yeah, any ETA, i'm looking to evaluate StencilJS before to adopt.
